coverage.sh

```diff
@@ -32,6 +32,15 @@ fi
 ./sizes
 ./constants
 ./multi
+for i in $(./hashsum | tail -n +3 | awk '{print $1}' | sort); do echo -n "$i: " && ./hashsum $i testprof/test.key ; done > hashsum_tv.txt
```

Comment:
I also thought about that, but as we have configurations like e.g. `LTC_EASY` which don't include all hashes I thought it's enough here.
But we could also move it to build.sh and add separate known-good testvectors for the different configurations.